packages/boot/test/fast-usdc/snapshots/fast-usdc.test.ts.md

```diff
@@ -18,6 +18,20 @@ Generated by [AVA](https://avajs.dev).
       ],
     ]
 
+## writes fee config to vstorage
+
+> Under "published", the "fastUsdc.feeConfig" node is delegated to the fee configuration for Fast USDC.
+> The example below illustrates the schema of the data published there.
+> 
+> See also board marshalling conventions (_to appear_).
+
+    [
+      [
+        'published.fastUsdc.feeConfig',
+        '{"blockHeight":"0","values":["{\\"body\\":\\"#{\\\\\\"contractRate\\\\\\":{\\\\\\"denominator\\\\\\":{\\\\\\"brand\\\\\\":\\\\\\"$0.Alleged: USDC brand\\\\\\",\\\\\\"value\\\\\\":\\\\\\"+10\\\\\\"},\\\\\\"numerator\\\\\\":{\\\\\\"brand\\\\\\":\\\\\\"$0\\\\\\",\\\\\\"value\\\\\\":\\\\\\"+2\\\\\\"}},\\\\\\"flat\\\\\\":{\\\\\\"brand\\\\\\":\\\\\\"$0\\\\\\",\\\\\\"value\\\\\\":\\\\\\"+10000\\\\\\"},\\\\\\"maxVariable\\\\\\":{\\\\\\"brand\\\\\\":\\\\\\"$0\\\\\\",\\\\\\"value\\\\\\":\\\\\\"+5000000\\\\\\"},\\\\\\"variableRate\\\\\\":{\\\\\\"denominator\\\\\\":{\\\\\\"brand\\\\\\":\\\\\\"$0\\\\\\",\\\\\\"value\\\\\\":\\\\\\"+100\\\\\\"},\\\\\\"numerator\\\\\\":{\\\\\\"brand\\\\\\":\\\\\\"$0\\\\\\",\\\\\\"value\\\\\\":\\\\\\"+1\\\\\\"}}}\\",\\"slots\\":[\\"board01029\\"]}"]}',
```

Comment:
odd... why are we getting a string of encoded gook here?

I would expect something more like [inter-protocol/test/vaultFactory/snapshots/vaultFactory.test.js.md](https://github.com/Agoric/agoric-sdk/blob/master/packages/inter-protocol/test/vaultFactory/snapshots/vaultFactory.test.js.md)

looking into it...

Reply:
The `storage` arg to `documentStorageSchema` is `MockChainStorageRoot | FakeStorageKit`. `MockChainStorageRoot` knows how to unmarshal, but we're providign a `FakeStorageKit`.